src/pages/Sign/sections/SignSubscription.tsx

```diff
@@ -93,51 +106,88 @@ const handleMessage = (
   return textkeys[message]
 }
 
-export const SubscriptionComponent: React.SFC = () => (
-  <Subscription<SignStatusData> subscription={SIGN_SUBSCRIPTION}>
-    {({ data, loading, error }) => {
-      if (loading || !data) {
-        return null
+interface StateComponentProps {
+  signState?: SIGNSTATE
```

Comment:
Sign state should never be null. https://github.com/HedvigInsurance/member-registry-service/blob/121314ae7cb9ecf3cac2eb694b517ef26520a43d/member-service/src/main/java/com/hedvig/memberservice/web/v2/dto/SignStatusResponse.java#L13

Reply:
(after irl discussion) sign no, state can be null (undefined) b/c `data` can be null

Reply:
if we haven't completed the query yet